### PR TITLE
kv: tiny clean up

### PR DESCRIPTION
**Pull request by zimulala**
Reduce twice judgments when the transaction is running normally and make `RunInNewTxn` more readable.

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: kv/txn.go [hunk 1]
```diff
@@ -38,35 +38,34 @@ func RunInNewTxn(store Storage, retryable bool, f func(txn Transaction) error) e
...
 
+		// originalTxnTS is used to trace the original transaction when the function is retryable.
 		if i == 0 {
 			originalTxnTS = txn.StartTS()
 		}
 
 		err = f(txn)
-		if retryable && IsRetryableError(err) {
-			log.Warnf("[kv] Retry txn %v original txn %v err %v", txn, originalTxnTS, err)
-			err1 := txn.Rollback()
```
Comment (zz-jason) [inline kv/txn.go]: why `txn.Rollback()` is removed ?

Comment (coocood) [reply kv/txn.go]: it's not removed, it's merged into `if err != nil` at line 47

**Pull request merged**